daprdocs/content/en/developing-applications/building-blocks/pubsub/pubsub-message-ttl.md

```diff
@@ -0,0 +1,37 @@
+---
+type: docs
+title: "Pub/Sub Message's Time-To-Live"
+linkTitle: "Pub/Sub Message's TTL"
+weight: 5000
+description: "Use time-to-live in Pub/Sub messages." 
+---
+
+## Introduction
+
+In Kafka, time-to-live is configured in the topic via `retention.ms` as per [documentation](https://kafka.apache.org/documentation/#topicconfigs_retention.ms). This setup is transparent to Dapr.
+
+On the other hand, Azure Service Bus supports [entity level time-to-live]((https://docs.microsoft.com/en-us/azure/service-bus-messaging/message-expiration)). It means that messages have a default time-to-live but also can be set with a shorter timespan at publishing time.
+
+Dapr enables per message time-to-live for all Pub/Sub components, falling back automatically to native support from the component when applicable. It means that applications can set time-to-live per message and subscribers will not receive those messages after expiration.
+
+### Pub/Sub Components with Message TTL
+
+* All, some are implemented natively.
```

Comment:
We either need to expand "some" into the full list here, or preferrably we add a table & column to https://v1-rc3.docs.dapr.io/operations/components/setup-pubsub/supported-pubsub/ similar to https://v1-rc3.docs.dapr.io/operations/components/setup-state-store/supported-state-stores/

Reply:
I've added "All" because this is the default behavior regardless of the component implementation. The list of components that have "native support" is shown in the list below. Otherwise, we would need to remember to add every new pubsub to this list. Having to add only the ones with "native support" is more reasonable.